run/o2sim_kine_publisher.cxx

```diff
@@ -0,0 +1,58 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#include "Framework/AnalysisTask.h"
+#include "Framework/runDataProcessing.h"
+#include "SimulationDataFormat/MCTrack.h"
+#include "Steer/MCKinematicsReader.h"
+
+using namespace o2::framework;
+using namespace o2::steer;
+
+struct kinepublisher {
```

Comment:
structs and classes should be capitalized according to conventions. Otherwise this looks fine.